### PR TITLE
Refactored AuthController & Auth routes.

### PR DESCRIPTION
**Pull request by fergthh**
### List of Changes:

- [x] Removed unnecessary package which does not allow overwriting Authentication routes.
- [x] Changed Trait used by AuthController (Thanks @RDelorier). 
- [x] Refactor AuthController (cleanup) & Auth routes.

### CODE REVIEW

File: app/Http/routes.php [hunk 1]
```diff
@@ -36,6 +36,14 @@
...
-Route::get('/auth/logout', 'Auth\AuthController@getLogout');
+$router->group([
+    'namespace' => 'Auth',
+    'prefix' => 'auth',
+], function(){
+    Route::get('login', 'AuthController@getLogin');
+    Route::post('login', 'AuthController@postLogin');
+    Route::get('logout', 'AuthController@getLogout');
+    Route::match(['get', 'post'], 'register', function(){
+        return redirect('/');
```
Comment (RDelorier) [inline app/Http/routes.php]: You could change the trait user by AuthController to AuthenticatesUsers and get rid of this redirect

Comment (fergthh) [inline app/Http/routes.php]: @RDelorier Yeah! You're right! Thanks for the comment! ;) Changed.

**Comment by austintoddj** [comment]
Great work @fergthh! One thought: I'd like to implement a feature in the near future that will allow the admin user to change their password from their profile setting page. Will your changes to the password controller affect that?

**Comment by fergthh** [comment]
@austintoddj Yes, it does. But I just fixed it.

**Comment by RDelorier** [comment]
@fergthh I opened a pr to your branch adding the password update feature @austintoddj mentioned if you want to merge it and include it with this pr then its ready for ya, or you can close it and I'll wait till this is merged.

**Comment by RDelorier** [comment]
I suppose I should add tests also...

**Pull request merged**